### PR TITLE
doc: fundchannel example

### PR DESCRIPTION
**Pull request by urza**
This should hopefully alleviate the pain of figuring out how to properly quote utxos in fundchannel command :) It took me a while to figure it out and I would be lost without @cdecker s help on IRC.

Changelog-None: Docs change only

### CODE REVIEW

File: doc/lightning-fundchannel.7.md [hunk 1]
```diff
@@ -64,6 +64,11 @@ unrecoverable once pushed.
 on close. Only valid if both peers have negotiated `option_upfront_shutdown_script`.
 Returns `close_to` set to closing script iff is negotiated.
 
+This example shows how to use lightning-cli to open new channel with peer 03f...fc1 from one whole utxo bcc1...39c:0
+(you can use **listfunds** command to get txid and vout):
+
+	lightning-cli fundchannel id=03f...fc1 amount=all feerate=normal utxos='["bcc1...39c:0"]'
```
Comment (jsarenik) [inline doc/lightning-fundchannel.7.md:70]: `lightning-cli -k` maybe?

**Review by vincenzopalazzo — APPROVED** [review]
ack https://github.com/ElementsProject/lightning/pull/4545/commits/2c741cd83530cfa8b39df67811a09db299720f72

Looks good to me.

File: doc/lightning-fundchannel.7.md (hunk 1 above)
Comment (urza) [reply doc/lightning-fundchannel.7.md]: Thanks, added.

**Comment by jsarenik** [comment]
ACK 900be617effc69c6ab716be20ff8157b3a11267e

@urza Still needs `Changelog-None` to make the changelog bot happy. See for example 65bf347ad.

**Comment by urza** [comment]
I added another file by accident when adding the -k. This is now fixed, so should be all good.

**Comment by jsarenik** [comment]
ACK a078e7fc5f4b9f563009d08d8609de4d3dbcdef3

**Review by jsarenik — APPROVED** [review]

**Pull request merged**